src/main/java/frc/robot/subsystems/swerve/modules/ModuleConstants.java

```diff
@@ -1,20 +1,34 @@
 package frc.robot.subsystems.swerve.modules;
 
 import edu.wpi.first.math.geometry.Rotation2d;
-import frc.robot.constants.LogPaths;
-import frc.robot.subsystems.swerve.SwerveConstants;
 import frc.utils.Conversions;
 import frc.utils.battery.BatteryUtils;
 
-public record ModuleConstants(double wheelDiameterMeters, double couplingRatio, Rotation2d velocityAt12VoltsPerSecond) {
+public record ModuleConstants(
+        String logPath,
+        String alertLogPath,
+        double wheelDiameterMeters,
+        double couplingRatio,
+        Rotation2d velocityAt12VoltsPerSecond
+) {
 
-    public ModuleConstants(double wheelDiameterMeters, double couplingRatio, double velocityAt12VoltsMetersPerSecond){
-        this(wheelDiameterMeters, couplingRatio, Conversions.distanceToAngle(velocityAt12VoltsMetersPerSecond, wheelDiameterMeters));
+    public ModuleConstants(
+            String swerveLogPath,
+            double wheelDiameterMeters,
+            double couplingRatio,
+            double velocityAt12VoltsMetersPerSecond) {
+        this(
+                swerveLogPath + "Modules/",
+                swerveLogPath + "Modules/Alerts/",
```

Comment:
no need to save alert path (i want in future to create more robust handling for alerts)

Reply:
so just use constants.logPath() + "Alerts/" in the code?

Reply:
LogPathsConstnats.Alerts + constants.logpath